docusaurus/docs/reactnative/basics/translations.mdx

```diff
@@ -52,18 +51,158 @@ export const App = () =>
   </OverlayProvider>;
 ```
 
-## Languages
+### Setting language for components
 
-The default language for `Streami18n` is English (en).
-Currently seven total languages are provided out of the box.
+Stream provides built in translations for some languages out-of-the-box.
+`Streami18n` accepts two optional parameters when being instantiated, [`options`](#options) and [`i18nextConfig`](#i18nextconfig).
+These parameters allow you to modify the `Streami18n` instance to your preferences.
 
-- English (en)
-- Dutch (nl)
-- Russian (ru)
-- Turkish (tr)
-- French (fr)
-- Italian (it)
-- Hindi (hi)
+As an example, let's say we need to localize the UI of the application for a Dutch audience:
+
+```tsx
+const streami18n = new Streami18n({ language: 'nl' }); // Instantiate Streami18n with Dutch strings.
+```
+
+Alternatively, you can also use [`setLanguage`](#setlanguage) method on `Streami18n` class.
+This is useful especially if you want to build language toggle functionality within your app.
+
+For example, let's say an application needs to default to English but support French:
+
+```tsx
```

Comment:
Trying to make it clear that the original instantiation and switching languages has Integrator defined logic that should go between them.

Reply:
Admittedly, I am assuming this is how it would work in practice. Let me know if I am off on this.